www/scripts/codecheckerviewer/ListOfBugs.js

```diff
@@ -248,6 +270,7 @@ function (declare, dom, Deferred, ObjectStore, Store, QueryResults, topic,
         { name : 'Message', field : 'checkerMsg', width : '100%', formatter : checkerMessageFormatter },
         { name : 'Checker name', field : 'checkerId', width : '50%', formatter: checkerNameFormatter },
         { name : 'Severity', field : 'severity', cellClasses : 'severity', width : '15%', formatter : severityFormatter },
+        { name : 'Bug path length', field : 'bugPathLength', cellClasses : 'bug-path-length', width : '15%', formatter : bugPathLengthFormatter },
```

Comment:
I think we should find a shorter name for this column. Isn't `Length` enough? When the header for the column is longer than the value printed, something's not quite right.

Reply:
I think `Difficulty` is much better. What do you think @whisperity?

Reply:
Sounds good to me.